index.js

```diff
@@ -1,88 +1,239 @@
-  }
+          'perfectionist/sort-named-imports': ['error', {
+            type: 'custom',
+            order: 'asc',
+            ignoreCase: false
+          }]
+        }
+      })
+      .override('antfu/vue/rules', {
+        rules: {
+          'vue/attributes-order': 'off',
+          'vue/comma-dangle': ['error', 'never'],
+          'vue/component-name-in-template-casing': ['warn', 'PascalCase'],
+          'vue/custom-event-name-casing': ['error', 'kebab-case'],
+          'vue/define-macros-order': 'off',
+          'vue/html-closing-bracket-newline': 'off',
+          'vue/no-extra-parens': 'off',
+          'vue/no-useless-v-bind': ['error', { ignoreStringEscape: true }],
+          'vue/no-v-text-v-html-on-component': 'off',
+          'vue/singleline-html-element-content-newline': 'off'
+        }
+      })
+      .override('antfu/node/rules', {
+        rules: {
+          'node/prefer-global/process': 'off'
+        }
+      })
+      // `antfu/imports/rules` got configured twice in the antfu config,
+      // Using only `.override` would not be enough, as the second one would override our changes.
+      // So we use `.append` to add our rules after the antfu config.
+      .append({
+        name: 'globalbrain/imports/rules',
+        rules: {
+          'import/consistent-type-specifier-style': ['error', 'inline'],
+          'import/no-duplicates': [
+            'error',
+            {
+              'prefer-inline': true
+            }
+          ]
+        }
+      })
+      .append({
+        name: 'globalbrain/release-it-config',
+        files: ['**/.release-it.{js,ts}'],
+        rules: {
+          'no-template-curly-in-string': 'off'
+        }
+      })
+      .append({
+        name: 'globalbrain/looser-stylistic-rules',
+        rules: {
+          // We don't actually care about these stylistic choices
+          'test/consistent-test-it': 'off',
+          'test/prefer-lowercase-title': 'off'
+        }
+      })
+      .append({
+        name: 'globalbrain/todo/new-rules-that-need-consensus',
+        rules: {
+          // Just new stylistic rules, mostly auto-fixable.
+          'antfu/consistent-list-newline': 'off',
+          'style/max-statements-per-line': 'off',
+          'style/member-delimiter-style': 'off',
+          'style/operator-linebreak': 'off',
+
+          // Didn't have these in the past.
+          'jsonc/sort-array-values': 'off',
+          'jsonc/sort-keys': 'off',
+          'toml/indent': 'off',
+          'toml/array-bracket-spacing': 'off',
+
+          // We have import sorting rules already,
+          // do we need export sorting, too?
+          'perfectionist/sort-exports': 'off',
+          'perfectionist/sort-named-exports': 'off',
+
+          // https://www.totaltypescript.com/method-shorthand-syntax-considered-harmful
+          // A lot of violations in our codebase
+          'ts/method-signature-style': 'off',
+
+          'unicorn/new-for-builtins': 'off'
```

Comment:
any reason to disable this? rule itself seems fine

Reply:
Was just trying to keep the number of new errors to a minimum. But yeah this seems fine and we've got very few violations. Let's enable it.